brave/src/main/java/com/linecorp/armeria/common/brave/RequestContextCurrentTraceContext.java

```diff
@@ -248,15 +291,29 @@ public String toString() {
      * Armeria code should always have a request context available, and this won't work without it.
      */
     @Nullable
-    private static RequestContext getRequestContextOrWarnOnce() {
+    private RequestContext getRequestContextOrWarnOnce() {
+        if (Boolean.TRUE.equals(THREAD_NOT_REQUEST_THREAD.get())) {
+            return null;
+        }
+        if (!nonRequestThreadPatterns.isEmpty()) {
+            final String threadName = Thread.currentThread().getName();
+            for (Pattern pattern : nonRequestThreadPatterns) {
+                if (pattern.matcher(threadName).matches()) {
```

Comment:
How about using `find()` instead of `matches()`, so that a user does not need to append `.*`?